R/dplyr.R

```diff
@@ -1,7 +1,7 @@
 src_monetdb <- function(dbname="demo", host = "localhost", port = 50000L, user = "monetdb",
   password = "monetdb", con = FALSE, ...) {
   if (!inherits(con, "MonetDBConnection") || !DBI::dbIsValid(con)) {
-    con <- DBI::dbConnect(MonetDB.R(), dbname = dbname , host = host, port = port,
+    con <- DBI::dbConnect(MonetDBLite(), dbname = dbname , host = host, port = port,
```

Comment:
made the first two changes

2016-05-25 8:38 GMT-04:00 Hannes Mühleisen <notifications@github.com>:

> In R/dplyr.R
> <https://github.com/hannesmuehleisen/MonetDBLite/pull/35#discussion_r64564437>
> :
>
> > @@ -1,7 +1,7 @@
> >  src_monetdb <- function(dbname="demo", host = "localhost", port = 50000L, user = "monetdb",
> >    password = "monetdb", con = FALSE, ...) {
> >    if (!inherits(con, "MonetDBConnection") || !DBI::dbIsValid(con)) {
> > -    con <- DBI::dbConnect(MonetDB.R(), dbname = dbname , host = host, port = port,
> > +    con <- DBI::dbConnect(MonetDBLite(), dbname = dbname , host = host, port = port,
>
> This should also stay a call to MonetDB.R(). We could add another
> function src_monetdblite() for this without all the dbname crap and a
> dbdir instead.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hannesmuehleisen/MonetDBLite/pull/35/files/951e99e227c24b01575723958d86f819089b6fa1#r64564437>
>